main.go

```diff
@@ -62,14 +62,28 @@ type proxyFlags struct {
 type etcdManager struct {
 	etcd.ServerConfig
 	logger             log.Logger
-	removeTimeout      int64
+	removeTimeout      time.Duration
 	unhealthyMemberTTL int64
 	operation          string
-	targetCluster      string
+	targetCluster      []string
 	server             *etcd.Server
 	client             *etcd.Client
 }
 
+func (mgr *etcdManager) setup(conf *config.ProxyConfig) {
+	mgr.LPAddress = *conf.ListenOnPeerAddress
+	mgr.APAddress = *conf.AdvertisePeerAddress
+	mgr.LCAddress = *conf.ListenOnClientAddress
+	mgr.ACAddress = *conf.AdvertiseClientAddress
+	mgr.MAddress = *conf.ETCDMetricsAddress
+	mgr.UnhealthyMemberTTL = *conf.UnhealthyMemberTTL
+	mgr.removeTimeout = *conf.RemoveMemberTimeout
+	mgr.DataDir = *conf.ClusterDataDir
+	mgr.Name = *conf.ClusterMemberName
+	mgr.operation = *conf.ClusterOperation
+	mgr.targetCluster = conf.TargetClusterAddresses
```

Comment:
can one specify in the config file the full list of machines in the cluster, then override the first one with --operation seed? if so, cool and all we need is the below comment. Does the first machine need to null this out with a commandline override?